### ISSUE TITLE
Pause / Resume not working properly

### ISSUE THREAD

**Issue by ReporterX** [tabwrangler/tabwrangler#267]
**To Reproduce**
1. Let's say you set it to auto close at 60 mins.
2. Pause in the middle of countdown. Tab 1 countdown is 45 mins left. Tab 2 countdown is 40 mins left.
3. Press resume

**Expected behavior**
The countdown continues. Tab 1 should start at 45 mins, tab 2 at 40 mins.

**Actual behavior**
All tabs start over again, starting at 60 mins.

If the actual behavior is by design, the wordings are misleading. I would suggest calling it as **"Stop" / "Start"**.

**Version (please complete the following information):**
 - OS: Windows
 - Browser: Firefox
 - Version: 6.5.1

**Comment by ssorallen** [comment]
Agreed, this could be improved; "Pause" is misleading. Is "Pause" the actual behavior you would want?

**Comment by KTamas** [comment]
This seems to be the behavior now? "All tabs start over again, starting at 60 mins."?

**Comment by ssorallen** [comment]
This is fixed in https://github.com/tabwrangler/tabwrangler/commit/e9a66c0f85600f696b4867786a5aabc473d8885f, and the next version (v8.2.0) will include the change.
